descriptors/ModuleDescriptor-template.json

```diff
@@ -58,7 +58,7 @@
   "optional": [
     {
       "id": "custom-fields",
-      "version": "2.1"
+      "version": "2.1 3.0"
     },
```

Comment:
also this setting can be removed from readme now
![image](https://github.com/user-attachments/assets/e3e4387c-c72a-4a95-a7f6-dc3a0bb378cc)